src/App.vue

```diff
@@ -1,15 +1,15 @@
 <template>
-    <div class="w-full min-h-screen px-1vw reset" :class="[theme]">
-        <NavBar />
-
+    <NavBar />
+    <div class="pp-base min-h-screen px-1vw reset" :class="[theme]">
         <router-view v-slot="{ Component }">
             <keep-alive :max="5">
                 <component :is="Component" :key="$route.fullPath" />
             </keep-alive>
         </router-view>
-
-        <FooterComponent />
     </div>
+    <label :class="[theme]">
```

Comment:
The `:class="[theme]"` is not needed here and for the `NavBar` not either, Efy doesn't use it since it handles theme differently than Piped did previously.